binderhub/templates/loading.html

```diff
@@ -13,6 +13,9 @@
 <div id="loader-text">
 	<p>Loading your Binder...</p>
 </div>
+<div id="loader-links">
+  <p class="text-center">New to Binder? Check out the <a target="_blank" href="https://mybinder.readthedocs.io/en/latest/">Binder Documentation</a> for more information</p>
```

Comment:
```suggestion
  <p class="text-center">New to Binder? Check out the <a target="_blank" href="https://mybinder.readthedocs.io/en/latest/">Binder Documentation (opens in new window)</a> for more information</p>
```
What is the current UX recommendation about `target=_blank`? Personally I find it annoying when sites force me to open something in a new tab, as I am perfectly able to click "open in new tab" when I mean to do that. I can see that for new users it would be good to open it in a new window by default though maybe "new" in this context isn't new to binder but new to using browsers?

Reply:
The main reason I did it here was because this is a temporary "loading" screen. So in a sense we know what users are expecting: they're expecting to be taken to a running Binder. But if they get redirected to a new page first, then that will never happen.